spec/factories/projects.rb

```diff
@@ -0,0 +1,10 @@
+FactoryGirl.define do
+  factory :project do
+    id { Faker::Number.between(1, 10) }
+    name { Faker::StarWars.droid }
+    description { Faker::StarWars.wookie_sentence }
+    total { Faker::Number.between(1, 100) }
+    time { Faker::Time.between(DateTime.now - 1, DateTime.now) }
```

Comment:
I really thing that we should use a fixed date in the future. It's only because of the between time may not give us enough time for the duration calculation.